tensorflow/python/data/util/structure_test.py

```diff
@@ -50,8 +50,7 @@
 # has been set up.
 #
 # TODO(jsimsa): Add tests for OptionalStructure and DatasetStructure.
-class StructureTest(test_base.DatasetTestBase, parameterized.TestCase,
-                    test_util.TensorFlowTestCase):
+class StructureTest(test_base.DatasetTestBase, parameterized.TestCase):
 
   # pylint: disable=g-long-lambda,protected-access
   @parameterized.named_parameters(
```

Comment:
This should be replace with using `combinations.generate`. Here is an example of how you can create a product of test combinations: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/data/kernel_tests/shuffle_test.py#L224-L227

Reply:
addressed in the commit.